applications/basic_networking_ping/CMakeLists.txt

```
@@ -13,4 +13,12 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+cmake_minimum_required(VERSION 3.20)
+project(basic_networking_ping LANGUAGES NONE)
+
 add_subdirectory(cpp)
+
+if(HOLOHUB_BUILD_PYTHON) # since using python wrapping
+  add_subdirectory(python)
+endif()
+
```

```suggestion
add_subdirectory(python)
```
I think HOLOHUB_BUILD_PYTHON is meant to be used for deciding whether or not to build the holohub python operators